src/api.ts

```diff
@@ -2892,6 +2892,16 @@ export const methods = {
       countFailedResponse(api_name, 'filter not found')
       return
     }
+    if (Array.isArray(inputFilter.address) && inputFilter.address.length > 100) {
+      callback({ code: -32000, message: 'Invalid address' }, null)
+      countFailedResponse(api_name, 'Invalid address')
+      return
+    }
+    if (inputFilter.topics.length > 200) {
```

Comment:
Should we not check if the topics object is of type 'array' as well?

Reply:
Similar to what has been done for the address array.

Reply:
Good call